modular/uploader/upload_task.go

```diff
@@ -21,6 +22,13 @@ import (
 	storagetypes "github.com/bnb-chain/greenfield/x/storage/types"
 )
 
+var (
+	RtyAttNum = uint(3)
+	RtyAttem  = retry.Attempts(RtyAttNum)
+	RtyDelay  = retry.Delay(time.Millisecond * 500)
+	RtyErr    = retry.LastErrorOnly(true)
```

Comment:
I think these variables should be non-exported.

Reply:
yes